rs/prep/src/internet_computer.rs

```diff
@@ -850,19 +883,58 @@ impl IcConfig {
                 None
             };
 
+            let dcr = DataCenterRecord {
+                id: name.to_string(),
+                region: match obj["region"].as_str() {
+                    Some(region) => region.to_string(),
+                    None => String::from(""),
+                },
+                owner: match obj["owner"].as_str() {
+                    Some(owner) => owner.to_string(),
+                    None => String::from(""),
+                },
```

Comment:
You could consider the simpler:
```suggestion
                region: obj["region"].as_str().unwrap_or("").to_string(),
                owner: obj["owner"].as_str().unwrap_or("").to_string(),
```

Reply:
Thanks Bas.

It's mostly muscle memory by now (and also a style that we adopted in UTOPIA).

I'm happy to take a second pass at this at a later point and simplify the whole reading. For now I'd like to go through the whole 9 yards and have this landed into UTOPIA (the process is long and tedious).